### ISSUE TITLE
allow me to select the profile i want to use for open the email

### ISSUE THREAD

**Issue by voidplayer** [davesteele/gnome-gmail#34]
firefox -P -no-remote to start a separate session for my email...

Could it be possible to open the email on one specific profile instead of the default profile?

**Comment by davesteele** [comment]
Not as currently written. GG uses the python  [webbrowser](https://docs.python.org/2/library/webbrowser.html#browser-controller-objects) module to work with the right browser. It would be possible modify the browser object's [remote-args](https://hg.python.org/cpython/file/78b84ae0b745/Lib/webbrowser.py#l247) to add the parameters, but that is browser-specific and uses undocumented features.

You realize that the profile used and the email account used are independent? The email account is prompted before launching the browser.

**Comment by voidplayer** [comment]
yeah i do realize. I dont use separate profiles for different mails, but because i dont want navigation data linked to my email :)

I also realize that this is browser specific. It would be nice to be able to pass parameters to the browser to be able to do this :)
